dev/tidb-in-kubernetes/reference/configuration/storage-class.md

```diff
@@ -83,6 +100,112 @@ For more information, refer to [Kubernetes local storage](https://kubernetes.io/
 
 Refer to [Best Practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) for more information on local PV in Kubernetes.
 
+## Instance
+
+For monitoring purposes, components like TiDB Binlog and backup use a local disk to storage data. Or you can mount the data into the corresponding directory and create different `StorageClass` in a SAS disk. Specifically:
```

Comment:
Fix it! Just realize I misunderstood the Chinese...

Reply:
Also, I've done a little search. It is more appropriate to use "a" instead of "an" in front of "SAS" because "SAS" is an acronym and pronounced as /sɑs/. Glad you point it out!